### ISSUE TITLE
About cls function.

### ISSUE THREAD

**Issue by LD421083** [Ascend-Research/HeadPoseEstimation-WHENet#5]
Hello, a sigmoid activation with binary cross entropy is selected for the classification loss function mentioned in your paper. If this is done, the classification part will become multilable classification, but multilable classification deals with the problem of non mutually exclusive multi classification. Angle classification belongs to the problem of mutually exclusive multi classification. I don't quite understand this concept. Can I have an answer? thank you!
